### PR TITLE
Small improvements to interactive examples

### PR DESCRIPTION
**Pull request by QuLogic**
## PR Summary

Adds a note about lack of interactivity in static docs, and tweaks some examples to look better once the multi-column sphinx-gallery CSS is removed.

## PR Checklist

**Tests and Styling**
- [n/a] Has pytest style unit tests (and `pytest` passes).
- [x] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).

**Documentation**
- [n/a] New features are documented, with examples if plot related.
- [n/a] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [n/a] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [x] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

File: examples/animation/pause_resume.py [hunk 1]
```diff
@@ -3,6 +3,14 @@
 Pausing and Resuming an Animation
 =================================
 
+.. note::
```
Comment (timhoffm) [inline examples/animation/pause_resume.py:6]: I would put the note below the description.

Comment (QuLogic) [reply examples/animation/pause_resume.py:6]: On all of them?

Where would you put it on the [interactive functions example](https://79-21548241-gh.circle-artifacts.com/0/doc/build/html/gallery/event_handling/ginput_manual_clabel_sgskip.html#sphx-glr-gallery-event-handling-ginput-manual-clabel-sgskip-py)?

Comment (timhoffm) [reply examples/animation/pause_resume.py:6]: Yes, on all of them. I have the feeling that we should first tell what the example is about and only after that discuss the technicalities in the note.

For the interactive functions, I'd place the note after the description sentence and right before the imports.

Comment (QuLogic) [reply examples/animation/pause_resume.py:11]: Done.

**Review by jklymak — APPROVED** [review]

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
Remilestoned to 3.6, because I'm not going to bother with a manual backport for this.

**Pull request merged**